apps/gitness/src/pages/repo/repo-summary.tsx

```diff
@@ -312,6 +332,8 @@ export const RepoSummaryList: React.FC = () => {
                     iconName: 'open-pr'
                   }
                 ]}
+                onAddDescription={() => {}}
```

Comment:
Not needed if onAddDescription is made optional

Reply:
fixed

Reply:
fixed